### PR TITLE
hash/ibm5170_cdrom.xml: add Star Trek games

### PR DESCRIPTION
**Pull request by chungy**
New working software list items (ibm5170_cdrom.xml)
---------------------------------------------------
Star Trek: 25th Anniversary (Enhanced CD-ROM) [Interplay]
Star Trek: Judgment Rites (Collector's Edition) [Interplay]

### CODE REVIEW

File: hash/ibm5170_cdrom.xml [hunk 1]
```diff
@@ -6169,6 +6169,49 @@ Includes DOS and Windows installation setup.
...
 
+	<!-- From the GOG release -->
+	<!-- Matches http://redump.org/disc/2431/ -->
+	<software name="strtrk25">
+		<description>Star Trek: 25th Anniversary (Enhanced CD-ROM)</description>
+		<year>1994</year>
+		<publisher>Interplay</publisher>
+		<info name="copy_protection" value="Manual required for map" />
+		<info name="developer" value="Interplay" />
+		<info name="language" value="English, French, German" />
```
Comment (ArcadeShadow) [inline hash/ibm5170_cdrom.xml:6180]: Please make the language info syntax like the others: 
<info name="language" value="English/French/German" />
For example, see the "Beneath a Steel Sky (Europe)" set.

**Comment by angelosa** [comment]
> Star Trek: 25th Anniversary (Enhanced CD-ROM) [Interplay]
Star Trek: Judgment Rites (Collector's Edition) [Interplay]

Please give context here: the square brackets are for crediting who dumped it. And I don't think it's necessary to specify Interplay here: it's implicit from the release names.

File: hash/ibm5170_cdrom.xml (hunk 1 above)
Comment (chungy) [reply hash/ibm5170_cdrom.xml]: > Please make the language info syntax like the others: `<info name="language" value="English/French/German" />` For example, see the "Beneath a Steel Sky (Europe)" set.

Thanks :)

**Comment by chungy** [comment]
> Please give context here: the square brackets are for crediting who dumped it. And I don't think it's necessary to specify Interplay here: it's implicit from the release names.

These are extracted out of the GOG.com release, so I think it's a bit unknown who dumped it. They match the noted redump.org entries, and I'd call that good enough.

**Pull request merged**